README.md

```diff
@@ -4,4 +4,7 @@ My blog hosted on Netlify: http://blog.shreyaspatil.dev/
 
 ## Credits
 
-This Blog is created using template provided by https://github.com/W3Layouts/gatsby-starter-delog
\ No newline at end of file
+This Blog is created using template provided by https://github.com/W3Layouts/gatsby-starter-delog
+
+## Here's how it looks
+![screenshot](/src/images/page.png)
```

Comment:
Can you please move this image in the root directory with a name - `screenshot.png`? It would be nice. Rest looks good 👍🏻

Reply:
thank you for the suggestion,
your suggested changes have been made !!
🙌🏻